app/controllers/ach_transfers_controller.rb

```diff
@@ -56,6 +56,16 @@ def create
     end
 
     if @ach_transfer.save
+      if params[:ach_transfer][:receipts]
+        receipts = params[:ach_transfer][:receipts].split ","
```

Comment:
This is odd. Can you look into a way that doesn't involve treating the input like a string?